src/customers/components/CustomerAddressListPage/CustomerAddressListPage.tsx

```diff
@@ -110,7 +110,12 @@ const CustomerAddressListPage: React.FC<CustomerAddressListPageProps> = props =>
               : intl.formatMessage(messages.noNameToShow)
           }
         >
-          <Button color="primary" variant="contained" onClick={onAdd}>
+          <Button
+            color="primary"
+            variant="contained"
+            onClick={onAdd}
+            data-test-id="add-address"
```

Comment:
Why are we using 2 different attribute names for the same thing? Let's try to use `data-test-id` everywhere

Reply:
I change data-test to data-test-id